pubspec.yaml

```diff
@@ -14,7 +14,7 @@ description: A new Flutter project.
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.3.0 <3.0.0"
```

Comment:
warum war das nötig?

Reply:
dann kann ich in array definitionen inline if und for verwenden (siehe https://github.com/shopwareLabs/product-import-app/pull/31/files#diff-54cfb0993820fad52b4bd7f81fb87168R221) :)

Reply:
Nice